### DISCUSSION TITLE
suppressHydrationWarning is not working on body tag (Next 14)

### DISCUSSION THREAD

**Discussion by nitishmittal1990** [vercel/next.js#68203] [Help]
### Summary

I want to suppress hydration warning in development mode, is there a way to enable this?

### Additional information

I am using Next `14.2.4`

**Comment by Juneezee** [comment]
See https://react.dev/reference/react-dom/client/hydrateRoot#suppressing-unavoidable-hydration-mismatch-errors. 

> This only works one level deep

You cannot surpress globally.

**Reply by nitishmittal1990** [reply]
Thanks @Juneezee, but is there a way to hide warnings in dev environment, something at config level?

**Reply by Juneezee** [reply]
I'm not sure what warnings are you getting.

As far as I know, `next.config.js` doesn't have an option to turn off warnings. The closest option might be `reactStrictMode`. You can try disabling React Strict Mode and see if that's what you wanted.

See more at https://nextjs.org/docs/app/api-reference/next-config-js/reactStrictMode
